src/view/Start.java

```diff
@@ -1,25 +1,37 @@
 package view;
 
-import java.awt.FlowLayout;
-
-import javax.swing.JButton;
 import javax.swing.JPanel;
+import javax.swing.SwingConstants;
+import javax.swing.JLabel;
+import javax.swing.JButton;
+
+import java.awt.Font;
+import java.awt.Color;
+import java.awt.GridLayout;
+import java.awt.BorderLayout;
 
 /**
  * 
  * Start ist ein view, welches JComponents
  * haelt und anbietet.
  * 
- * @author nina
+ * @author nina, niklas e.
  *
  */
 public class Start {
 	private JPanel content;
-
-	private JButton first;
-	private JButton second;
-	private JButton third;
-
+	
+	private JLabel titleLabel;
+	private JButton startButton;
+	private JButton optionsButton;
+	private JButton exitButton;
+	private JPanel gridPanel;
+	
+	private final Font ARIAL_BOLD_100 = new Font("Arial", Font.BOLD, 100);
+	private final Font ARIAL_PLAIN_50 = new Font("Arial", Font.PLAIN, 50);
+	private final Color STANDARD_COLOR = Color.CYAN;
+	private final Color TRANSPARENT_COLOR = new Color(0,0,0,0);
```

Comment:
Man hätte auch eine Style Klasse/ Interface  definieren können, die im Konstruktor des Views übergeben werden könnte. So könnte man in der main den Style des kompletten Programms anpassen.